en/docs/get-started/api-manager-quick-start-guide.md

```diff
@@ -112,7 +123,7 @@ Follow the instructions below to create and publish an API via the Publisher Por
      <tr> 
      <th>Endpoint
      </th>
-     <td><code>http://mock-json-server-url/</code>
+     <td><code>https://run.mocky.io/v3</code>
       <div class="admonition note">
       <p class="admonition-title">Note</p>
       <p><b>Use the HTTP protocol</b> because to use HTTPS you need to import the mock json server's certificate into WSO2 API-M.</p>
```

Comment:
I believe this comment is not correct. Mocky is using Lets Encrypt - I used HTTPs and it works perfectly.